packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -46,6 +46,8 @@ let MONGODB_USE_ENTERPRISE =
 
 // should we test compass-web (true) or compass electron (false)?
 export const TEST_COMPASS_WEB = process.argv.includes('--test-compass-web');
+export const ATLAS_DOMAIN = process.env.ATLAS_DOMAIN;
```

Comment:
For all these new variables you're adding, can we make the names more distinct to indicate the use-case better? Especially the CONNECTION_ ones, a lot of things rely on these, so just allowing arbitrary overrides without context is not great

Reply:
Sure, do you have some concrete suggestions? 

For the connection_ ones in particular, do you just mean adding ATLAS_ to the name or something

Reply:
Yeah, a common prefix we can stick on all of them is probably the way to go, something like `ATLAS_CLOUD_EXTERNAL_URL_TESTING_`, just so it's clear it's specifically for this "testing a deployed atlas cloud" scenario, not a generic way to control these variables

Reply:
Or rather `TEST_ATLAS_CLOUD_EXTERNAL_URL_` to match existing ones